client/src/components/Gallery.jsx

```diff
@@ -93,25 +93,25 @@ class Gallery extends React.Component {
     this.getImages();
   }
 
-  getImages(deal_id = 22) {
+  getImages() {
+    const dealId = window.location.pathname.split('/')[1];
```

Comment:
if we will be using the common url 'localhost:3000/deals/1 split index of id will be 2 not 1